storage/postgres/postgres.go

```diff
@@ -108,6 +108,40 @@ func (p *Postgres) Close(ctx context.Context) error {
 	return nil
 }
 
+func (p *Postgres) ReadRelationshipTuples(
+	ctx context.Context,
+	filter storage.ReadRelationshipTuplesFilter,
+	opts ...storage.QueryOption,
```

Comment:
This seems to be unused?

Reply:
i'm not sure what the intention was for this

Reply:
The goal is to use it to provide limits and grouping/aggregate functions. The goal is to have `ReadRelationshipTuples` generically replace all of the specific `ReadUserTuple`, `Read`, and `ReadUsersetTUples` etc.. We don't need to bloat our storage interface with all of those methods.

A single ReadRelationshipTuples method with optional filters and other query options is more elegant IMO.